mgmt/src/models/internal/device/settings.rs

```diff
@@ -23,28 +23,19 @@ pub enum PacketDriver {
 
 #[derive(Clone, Debug)]
 pub struct DeviceSettings {
-    hostname: Option<String>,
+    hostname: String,
```

Comment:
Why make it mandatory?

Reply:
It simplifies the code and we can always default it to something if needed.